lsp/server/src/utils/workspaceUtils.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import { SFDX_PROJECT_FILE } from './constants';
+import * as fs from 'fs';
+import * as path from 'path';
+import { WorkspaceFolder } from 'vscode-languageserver';
+
+export class WorkspaceUtils {
+    private static workspaceFolders: WorkspaceFolder[] | undefined | null = [];
+
+    static readonly DEFAULT_APP_PATH = path.join(
+        'force-app',
+        'main',
+        'default'
+    );
+
+    static readonly LWC_PATH = path.join(
+        WorkspaceUtils.DEFAULT_APP_PATH,
+        'lwc'
+    );
+
+    public static setWorkSpaceFolders(
+        workSpaceFolders: WorkspaceFolder[] | null | undefined
+    ) {
+        this.workspaceFolders = workSpaceFolders;
+    }
```

Comment:
`workSpaceFolders` shouldn't be undefined but either an array of `WorkspaceFolder` or `null`. Let's get rid of `undefined` here and in line 14.

```
export interface WorkspaceFoldersInitializeParams {
    /**
     * The workspace folders configured in the client when the server starts.
     *
     * This property is only available if the client supports workspace folders.
     * It can be `null` if the client supports workspace folders but none are
     * configured.
     *
     * @since 3.6.0
     */
    workspaceFolders?: WorkspaceFolder[] | null;
}
```